HLTrigger/Egamma/src/HLTElectronPFMTFilter.cc

```diff
@@ -1,4 +1,5 @@
 #include "HLTrigger/Egamma/interface/HLTElectronPFMTFilter.h"
+#include "HLTrigger/HLTcore/interface/defaultModuleLabel.h"

```

Comment:
defaultModuleLabel.h -> DefaultModuleLabel.h
(and the same in all other #include's)